src/screens/SettingsScreen.js

```diff
@@ -56,16 +57,23 @@ const TOP_FILTER = {
   LIST_TYPES: 'listTypes'
 };
 
-const INITIAL_FILTER = [
-  { id: TOP_FILTER.GENERAL, title: texts.settingsTitles.tabs.general, selected: true },
-  { id: TOP_FILTER.LIST_TYPES, title: texts.settingsTitles.tabs.listTypes, selected: false },
-  { id: TOP_FILTER.AR_DOWNLOAD_LIST, title: texts.settingsTitles.tabs.arSettings, selected: false }
-];
+const INITIAL_FILTER = (isARSupported) => {
+  return [
+    { id: TOP_FILTER.GENERAL, title: texts.settingsTitles.tabs.general, selected: true },
+    { id: TOP_FILTER.LIST_TYPES, title: texts.settingsTitles.tabs.listTypes, selected: false },
+    isARSupported && {
+      id: TOP_FILTER.AR_DOWNLOAD_LIST,
+      title: texts.settingsTitles.tabs.arSettings,
+      selected: false
+    }
+  ];
+};
```

Comment:
this results in

```
[{ id: TOP_FILTER.GENERAL, title: texts.settingsTitles.tabs.general, selected: true },
{ id: TOP_FILTER.LIST_TYPES, title: texts.settingsTitles.tabs.listTypes, selected: false },
false]
```

if AR is not supported. having the `false` is not the best solution. can you find a way to return the array with two elements if AR is not supported and alls three elements if it is supported?

Reply:
I don't think it's exactly what you wanted. but I tried something.

https://github.com/ikuseiGmbH/smart-village-app-app/pull/593/commits/735aa35dda1b5631991b68760609d245dd566f5e